### PR TITLE
Update SocialMedia.js

### PR DESCRIPTION
**Pull request by EZRankings-devs**

### CODE REVIEW

File: src/components/ContactUs/SocialMedia.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export const SocialIcon = (props) => {
       href={props.socialLink}
       target="_blank"
       rel="noreferrer"
+      rel="nofollow"
```
Comment (KimeraMoses) [inline src/components/ContactUs/SocialMedia.js:17]: We can not have multiple attributes on a JSX element, Please eliminate one of the two attributes of rel.

Comment (EZRankings-devs) [reply src/components/ContactUs/SocialMedia.js:17]: I have removed the "noreferrer' element.

Comment (KimeraMoses) [reply src/components/ContactUs/SocialMedia.js:17]: But I'm not seeing any new commit to this PR, Please check well and make sure it's removed and push again to this very pr

Comment (EZRankings-devs) [reply src/components/ContactUs/SocialMedia.js:17]: Okay sure

Comment (EZRankings-devs) [reply src/components/ContactUs/SocialMedia.js:17]: I have changed the attributed, Please check it.

image

**Pull request closed without merging**